<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainView</class>
 <widget class="QMainWindow" name="MainView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>571</width>
    <height>733</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainView</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="gbxWriteBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>551</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string>File Write</string>
    </property>
    <widget class="QPushButton" name="btnWrite">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>160</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::TabFocus</enum>
     </property>
     <property name="text">
      <string>Write</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblFileLocation">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>File Location</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblTextInput">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Text Input</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lneWriteFile">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>60</y>
       <width>521</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lneInputText">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>130</y>
       <width>521</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gbxFileRead">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>209</y>
      <width>551</width>
      <height>481</height>
     </rect>
    </property>
    <property name="title">
     <string>File Read</string>
    </property>
    <widget class="QPushButton" name="btnRead">
     <property name="geometry">
      <rect>
       <x>439</x>
       <y>30</y>
       <width>101</width>
       <height>24</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::TabFocus</enum>
     </property>
     <property name="text">
      <string>Read</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="plainTextEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>531</width>
       <height>391</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="lneReadFile">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>33</y>
       <width>411</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>571</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>lneWriteFile</tabstop>
  <tabstop>lneInputText</tabstop>
  <tabstop>btnWrite</tabstop>
  <tabstop>lneReadFile</tabstop>
  <tabstop>btnRead</tabstop>
  <tabstop>plainTextEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
 <slots>
  <slot>m_write_button_pushed()</slot>
  <slot>m_read_button_pushed()</slot>
 </slots>
</ui>
